### ISSUE TITLE
Examples of NeuralNetworks(Perceptron, MultilayerPerceptron) will be helpful

### ISSUE THREAD

**Issue by mdeora** [php-ai/php-ml-examples#4]
I would be great if more examples of NeuralNetwork classes eg. MultiLayerPerceptron or using BackPropagation algo are created with examples.
May be Below link example can be implementated -
https://ujjwalkarn.me/2016/08/09/quick-intro-neural-networks/

**Comment by itcraftsmanpl** [comment]
Yes, more examples are needed, but I have plans to create separate repository for neural network.

**Comment by jinaddavid** [comment]
please is there any link to any tutorial for easy understanding

**Comment by akondas** [comment]
http://php-ml.readthedocs.io/en/latest/machine-learning/neural-network/multilayer-perceptron-classifier/

**Comment by sanzzjose** [comment]
Please, could you provide an implementation of the methods trainsample and predictsample? thank you
